core/src/main/scala/org/locationtech/rasterframes/util/package.scala

```diff
@@ -183,6 +184,27 @@ package object util {
     }
   }
 
+  implicit class DFWithPrettyPrint(val df: Dataset[_]) extends AnyVal {
+    def toMarkdown(numRows: Int = 5, truncate: Boolean = false): String = {
+      import df.sqlContext.implicits._
+      val cols = df.columns
+      val header = cols.mkString("| ", " | ", " |") + "\n" + ("|---" * cols.length) + "|\n"
+      val stringifiers = cols
+        .map(c => s"`$c`")
+        .map(c => df.col(c).cast(StringType))
+        .map(c => if (truncate) substring(c, 1, 40) else c)
```

Comment:
I think if a field is getting truncated it would be good to have an indicator. Example from the getting-started.pymd (as rendered)

`[+proj=sinu +lon_0=0 +x_0=0 +y_0=0 +a=63` is how it renders but
`[+proj=sinu +lon_0=0 +x_0=0 +y_0=0 +a...` would be clearer